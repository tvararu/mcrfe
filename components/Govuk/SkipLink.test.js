import renderer from "react-test-renderer";
import SkipLink from "./SkipLink";

describe("SkipLink", () => {
  it("renders correctly", () => {
    const component = renderer.create(<SkipLink />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
