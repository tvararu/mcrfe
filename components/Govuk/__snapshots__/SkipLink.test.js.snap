// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkipLink renders correctly 1`] = `
<a
  className="govuk-skip-link"
  href="#main-content"
>
  Skip to main content
</a>
`;
